src/PowerShellEditorServices/Server/PsesLanguageServer.cs

```diff
@@ -32,10 +32,16 @@ internal class PsesLanguageServer
         /// <summary>
         /// Create a new language server instance.
         /// </summary>
+        /// <remarks>
+        /// This class is only ever instantiated via <see
+        /// cref="EditorServicesServerFactory.CreateLanguageServer"/>. It is essentially a
```

Comment:
I'd try to keep XML comment elements on the same line where possible, just so it's easier to pick up for the reader.

Reply:
Just using a rewrap extension. If you know how to get VS Code rewrap to not break tags, that'd be great! Otherwise I'm more concerned with getting the documentation right and the website updated.